cmd/ocm-backplane/console/console.go

```diff
@@ -249,6 +247,11 @@ func (o *consoleOptions) run(cmd *cobra.Command, argv []string) error {
 	if err != nil {
 		return err
 	}
+	// If this is a test scenario, we throw an error as opposed of keeping this function in a infinite loop
+	if len(argv) != 0 && argv[0] == "console_GO_test" {
+		return fmt.Errorf("test concluded sucessfully")
```

Comment:
Basically, It's not recommended to add unit testing support code within the prod code. If prod code needs `console_GO_test` arg for unit testing means the actual code is not unit testable. 

May be something missing with function/component isolations.